kafka-connect-redis/src/main/scala/com/datamountaineer/streamreactor/connect/redis/sink/config/RedisSinkSettings.scala

```diff
@@ -51,15 +50,16 @@ object RedisSinkSettings {
     // Get the error-policy, num-of-retries, redis-connection-info
     val errorPolicy = config.getErrorPolicy
     val nbrOfRetries = config.getNumberRetries
+
     // Get the aliases
-    val aliases = config.getFieldsAliases().toSeq
+    val aliases = config.getFieldsAliases()
```

Comment:
why the .toSeq would make the difference. the aliases is a set.

Reply:
ok, i shall revert this